LDM-732.tex

```diff
@@ -33,12 +33,7 @@
 \setDocDate{\today}
 
 \setDocAbstract {
-The Vera C Rubin Observatory Network Verification Document (VNVD) and associated JIRA V\&V
-Project define the flow-down of specifications from higher level documents to the LSST
-Observatory Network (as defined in LSE-78 LSST Observatory Network), and the methods and
-resources that will be used to verify that the networks have met the specifications satisfactory
-for accepting the Summit Network into DM Subsystem Integration Test (DMSSIT) and LSST
-System Integration Test (SIT)
+The Vera C Rubin Observatory Network Verification Document (VNVD) and associated JIRA V\&V Project define the flow-down of specifications from higher level documents to the LSST Observatory Network (as defined in \citeds{LSE-78} LSST Observatory Network), and the methods and resources that will be used to verify that the networks have met the specifications satisfactory for accepting the Summit Network into DM Subsystem Integration Test (DMSSIT) and LSST System Integration Test (SIT)
```

Comment:
LSE-78 is a design document that contains a narrative on "Requirements' but no formal requirements specifications.  It also states on P4 : "The above requirements are documented in Rubin Observatory System Specifications (OSS, LSE-30), and they are flowed down to Rubin Observatory subsystem requirements documents, e.g. Data Management System Requirements (LSE-61)."   I would change your text to gove LSE-61 as the formal requirements document, not LSE-78. Section 2.1 in LSE-78 seems to be no more than a narrative summarising what is in the OSS and DMSR, and does not present requirements that can be formally verified.

Reply:
Agreed.  I'm also trying to figure out the document handles for DMSSIT and SIT.

Reply:
This is the only file in docushare that turns up when searching for DMSSIT...

Reply:
We don't use that dochandle. What are you looking for? For SITCom is it sitcomtn but there is not higher level handle for SITCOM. Most SITCOM docs are LSE

Reply:
I vote for removing the references to DMSSIT and SIT.